### PR TITLE
-Merge me- Genetics Return to form -Merge me-

### PR DESCRIPTION
**Pull request by hex37**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
Fix energy shields to use the right tech tree
Adds the dwarfism mutation
Adds the unbalanced mutation
Adds the Reaper Spider cloning recipe
Adds the Dream Daemon cloning recipe
Adds spider friend cloning mutation to egg-laying spiders
Changes how the wasonce (And its kids) interact with synths when they eat them. (It crushes them instead now)
Fixes the design disk being unable to print mutagenic injectors.
Adds the Gold Roach (Sprite courtesy of @TripleZ#1242 )
Adds the Gold Roachling (Sprite courtesy of @TripleZ#1242 )
Adds the Golden roach egg.
Adds Obese Kampfers, they eat snacks to reproduce!
Added a mutation recipe combo for creating and cloning gold roaches.
Added the Small Mutagenic Scanner (Sprite courtesy of @TripleZ#1242 )
Added Genetics Poster (Sprite courtesy of @Aeger Medela#2891 )
Added randomized blurb system to the Genetics poster for it to give random tips. It doesn't work, but that's okay because it doesn't break either.
Added an informational blurb for every. Single. Mutation recipe.
Added mutations to the Schlorgo (who is a mess),
Added mutations to the Diyaab 
Added mutations to the  Fennec / Fennix
MUTATION_RAND nerfed to 100 points
Added Brown Bear genetics recipe (thanks to @Aeger Medela#2891 )
Added Polar Bear genetics recipe (thanks to@Aeger Medela#2891 )
Added Bat genetics recipe (thanks to @Aeger Medela#2891 )
Added Sargoyle recipe (thanks to @Aeger Medela#2891 )
Added Giant Snake recipe (thanks to @Aeger Medela#2891 )
Added Possum recipe (thanks to @Aeger Medela#2891 )
Limb Efficiency equation heavily modified to be subject to diminishing returns from intense organ stacking
Player speed now scales off of limb efficiency instead of muscle efficiency
Modified robot limbs to have twice as many muscles as usual, to offset them not having blood vessels with the new equations
Dropped Exalt nerves to 120% organ efficiency
/:cl:

### CODE REVIEW

**Comment by hex37** [comment]
I'm gonna test this some more tomorrow.

**Comment by r4d6** [comment]
Yay. Smol Mutation!

**Comment by r4d6** [comment]
The `seuche_gold` icon_state seems to flicker in the icon change preview.

File: code/modules/mob/living/resize.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@ var/const/RESIZE_GIGANTOR = 1.5
 var/const/RESIZE_BIG = 1.15
 var/const/RESIZE_NORMAL = 1
 var/const/RESIZE_SMALL = 0.85
+var/const/RESIZE_POCKET_PRINCE = 0.7
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/resize.dm:7]:
```suggestion
var/const/RESIZE_POCKET_PRINCE = 0.75
```
?

Comment (r4d6) [reply code/modules/mob/living/resize.dm:7]: It make sense since the other sizes is increased/decreased by 0.15

**Comment by LongSleeves** [comment]
Happy to see my poster made it in! Hopefully we can get the poster working. Got ideas of where ya plan to put it yet?

File: maps/CEVEris/_Nadezhda_Colony_Omni_STU.dmm [hunk 2]
```diff
@@ -99641,6 +99638,12 @@
 /obj/random/pouch/low_chance,
 /turf/simulated/floor/tiled/techmaint,
 /area/nadezhda/maintenance/undergroundfloor1north)
+"wCi" = (
+/obj/structure/sign/genetics{
+	step_y = 0
```
Comment (Trilbyspaceclone) [inline maps/CEVEris/_Nadezhda_Colony_Omni_STU.dmm:99643]:
```suggestion
	pixel_y= 0
```

Comment (Trilbyspaceclone) [reply maps/CEVEris/_Nadezhda_Colony_Omni_STU.dmm:99643]: step brakes everything

**Comment by hex37** [comment]
> The `seuche_gold` icon_state seems to flicker in the icon change preview.

its okay it shouldn't flicker in game. As far as I could tell, at least

File: code/modules/genetics/recipes/cloning_recipes.dm [hunk 3]
```diff
@@ -206,9 +223,71 @@
...
+	return new /datum/genetics/mutation/copy_mob(/mob/living/simple_animal/hostile/giant_snake, "giant snake")
+
+//======================================================================
+//mouse to possom
+/datum/genetics/mutation_recipe/combination/mouse
+	required_on_clone_types = list(/mob/living/simple_animal/mouse)
+	required_mutations = list(MUTATION_GIGANTISM)
+
+/datum/genetics/mutation_recipe/combination/mouse/get_result()
+	return new /datum/genetics/mutation/copy_mob(/mob/living/simple_animal/opossum, "opossom")
```
Comment (r4d6) [inline code/modules/genetics/recipes/cloning_recipes.dm:293]: The lack of an endline is annoying me.

File: code/modules/organs/internal/muscles.dm [hunk 4]
```diff
@@ -16,6 +16,8 @@
 	icon_state = "robotic_muscle"
 	desc = "Expand and contract"
 	nature = MODIFICATION_SILICON
+	organ_efficiency = list(OP_MUSCLE = 200)
```
Comment (Trilbyspaceclone) [inline code/modules/organs/internal/muscles.dm:19]: Why are we making robots and the robotic printer better then the upgraded human ones?

File: code/modules/genetics/recipes/cloning_recipes.dm (hunk 3 above)
Comment (hex37) [reply code/modules/genetics/recipes/cloning_recipes.dm:293]: why though

**Pull request merged**